src/providers/hooks/useObservations.js

```diff
@@ -34,12 +39,19 @@ const useObservations = ( ): Object => {
 
     // When querying a realm to find objects (e.g. realm.objects('Observation')) the result we get back
     // and the objects in it are "live" and will always reflect the latest state.
-    const localObservations = realm.objects( "Observation" );
+
+    const localObservations = realm.objects( "Observation" ).sorted( "timeObservedAt", true );
+    console.log( localObservations.length, "local obs in useObservations" );
+    const notUploadedObs = realm.objects( "Observation" ).filtered( "timeSynced == null && timeUpdatedLocally != null" );
```

Comment:
Don't you just need `timeSynced == null` here? How can `timeUpdatedLocally` be null and if it can, why would you exclude those from upload?